src/AfriCC/EPP/Validator.php

```diff
@@ -273,6 +273,30 @@ class Validator
     ];
 
     /**
+     * @var array
+     */
+    protected static $contact_role = [
```

Comment:
does it make sense to work here with integer values? Why not work directly with strings?